app/assets/stylesheets/components/_page_banner.scss

```diff
@@ -0,0 +1,22 @@
+.app-phase-banner .govuk-tag {
+  color: govuk-colour("white");
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-weight: 700;
```

Comment:
I always get a bit anxious when I see specific values in css. Generally difficult to maintain.

I think for the font-weight, you can use:
`$govuk-font-weight-bold`, but I don't see a variable for letter spacing.

Reply:
Good point, I'll change it